packages/adapter-nuxt/public/Hero/javascript.vue

```diff
@@ -0,0 +1,225 @@
+<script setup>
+import { isFilled } from "@prismicio/client";
+
+defineProps(getSliceComponentProps(["slice", "index", "slices", "context"]));
+</script>
+
+<template>
+	<section
```

Comment:
You don't want to set the same 2 spaces indent as we did for Next?

Reply:
<img width="629" alt="Screenshot 2023-09-27 at 14 59 42" src="https://github.com/prismicio/slice-machine/assets/6568065/b9b55ead-6916-480d-ad76-e835a2611a8e">

I still very unclear about this, they're displayed correctly in my editor